packages/katello/katello/katello-service

```diff
@@ -74,17 +73,12 @@ def manage_services(action)
 
   services_by_priority(action == 'stop').each do |service|
     if service_exists?(service)
-      case action
-      when 'enable', 'disable'
-        puts "systemctl #{action} #{service}.service"
-        puts `systemctl #{action} #{service}.service`
-      else
-        puts "#{COMMAND} #{service} #{action}"
         puts `#{COMMAND} #{service} #{action}`
-      end
+        puts `#{COMMAND} #{service} #{action}`
```

Comment:
Actually I think this code was already correct. I didn't notice `COMMAND` was already `service-wait` rather than `/sbin/service`